### ISSUE TITLE
Failed to activate the omnisharp-atom package

### ISSUE THREAD

**Issue by ronrat** [OmniSharp/omnisharp-atom#208]
Attempting to update Omnisharp Package

**Atom Version**: 0.196.0
**System**: Microsoft Windows 8.1 Pro
**Thrown From**: [omnisharp-atom](https://github.com/OmniSharp/omnisharp-atom) package, v0.4.10

### Stack Trace

Failed to activate the omnisharp-atom package

```
At spacePen is not defined

ReferenceError: spacePen is not defined
  at Object.<anonymous> (C:\Users\rratzlaff\.atom\packages\omnisharp-atom\lib\omnisharp-atom\views\tooltip-view.ts:47:4)
  at Module._compile (module.js:452:26)
  at Object.loadFile [as .ts] (C:\Users\rratzlaff\AppData\Local\atom\app-0.196.0\resources\app.asar\src\typescript.js:99:19)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Object.<anonymous> (C:\Users\rratzlaff\.atom\packages\omnisharp-atom\lib\omnisharp-atom\features\lookup.ts:5:19)
  at Module._compile (module.js:452:26)
  at Object.loadFile [as .ts] (C:\Users\rratzlaff\AppData\Local\atom\app-0.196.0\resources\app.asar\src\typescript.js:99:19)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at new Feature (C:\Users\rratzlaff\.atom\packages\omnisharp-atom\lib\omnisharp-atom\omnisharp-atom.ts:16:20)
  at C:\Users\rratzlaff\.atom\packages\omnisharp-atom\lib\omnisharp-atom\omnisharp-atom.ts:79:72
  at arrayMap (C:\Users\rratzlaff\.atom\packages\omnisharp-atom\node_modules\lodash\index.js:1449:25)
  at Function.map (C:\Users\rratzlaff\.atom\packages\omnisharp-atom\node_modules\lodash\index.js:6769:14)
  at OmniSharpAtom.loadFeatures (C:\Users\rratzlaff\.atom\packages\omnisharp-atom\lib\omnisharp-atom\omnisharp-atom.ts:79:26)
  at OmniSharpAtom.activate (C:\Users\rratzlaff\.atom\packages\omnisharp-atom\lib\omnisharp-atom\omnisharp-atom.ts:47:34)
  at Package.module.exports.Package.activateNow (C:\Users\rratzlaff\AppData\Local\atom\app-0.196.0\resources\app.asar\src\package.js:222:19)
  at C:\Users\rratzlaff\AppData\Local\atom\app-0.196.0\resources\app.asar\src\package.js:203:30
  at Package.module.exports.Package.measure (C:\Users\rratzlaff\AppData\Local\atom\app-0.196.0\resources\app.asar\src\package.js:147:15)
  at Package.module.exports.Package.activate (C:\Users\rratzlaff\AppData\Local\atom\app-0.196.0\resources\app.asar\src\package.js:195:14)
  at PackageManager.module.exports.PackageManager.activatePackage (C:\Users\rratzlaff\AppData\Local\atom\app-0.196.0\resources\app.asar\src\package-manager.js:434:21)
  at C:\Users\rratzlaff\AppData\Local\atom\app-0.196.0\resources\app.asar\node_modules\settings-view\lib\package-manager.js:270:29
  at exit (C:\Users\rratzlaff\AppData\Local\atom\app-0.196.0\resources\app.asar\node_modules\settings-view\lib\package-manager.js:41:16)
  at triggerExitCallback (C:\Users\rratzlaff\AppData\Local\atom\app-0.196.0\resources\app.asar\src\buffered-process.js:213:47)
  at ChildProcess.<anonymous> (C:\Users\rratzlaff\AppData\Local\atom\app-0.196.0\resources\app.asar\src\buffered-process.js:235:18)
  at emitTwo (events.js:87:13)
  at ChildProcess.emit (events.js:169:7)
  at Process.ChildProcess._handle.onexit (child_process.js:1057:12)

```

### Commands

```
     -4:19.6.0 core:copy (atom-text-editor.editor)
     -4:12.1.0 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "autocomplete",
      "autocomplete-plus-async"
    ],
    "autoHideMenuBar": true,
    "projectHome": "c:\\code"
  }
}
```

### Installed Packages

```coffee
# User
atom-yeoman, v0.2.0
autocomplete-plus, v2.12.1
color-picker, v1.7.0
jsonlint, v1.1.1
language-csharp, v0.5.0
language-mediawiki, v0.1.0
language-powershell, v2.0.1
language-typescript, v0.15.0
linter, v0.12.1
minimap, v4.8.0
omnisharp-atom, v0.4.10
project-manager, v1.15.5

# Dev
No dev packages
```

**Comment by nosami** [comment]
Thanks for reporting. Should be fixed in v0.4.12. Please re-open if it isn't.

**Comment by jchannon** [comment]
I just updated to 0.4.33 and got this error

**Comment by luisrudge** [comment]
I'm having this error too

**Comment by ferdbold** [comment]
Just got redirected to this issue while updating to 0.5.3

**Comment by bitbonk** [comment]
Also got redirected to this issue while updating to 0.5.3

**Comment by nosami** [comment]
Can't reproduce this.

**Comment by ladislaolainez** [comment]
Same here after updating to 0.5.5.

**Comment by nosami** [comment]
I just wiped all my packages from `~/.atom/packages` and reinstalled omnisharp-atom and it's working.

**Comment by wmeints** [comment]
Looks like an issue with the update or something very old that causes the plugin to crash. I had the same issue, removing and reinstalling worked. +1

**Comment by david-driscoll** [comment]
It's possible that if you're running solution and updating at the same time that there are locked files causing a problem?  I know atom typescript has a similar issue.

Is that the case here?  If it is, we can probably attempt to shutdown any active servers when atom announces its updating us.

**Comment by sgwill** [comment]
Just repro again after removing and reinstalling.

**Comment by sedonico** [comment]
same problem has occurred. after removing and reinstalling.

**Comment by david-driscoll** [comment]
Hopefully fixed by #431
